codec_packetizers/h264/h264_depacketizer.c

```diff
@@ -254,7 +248,10 @@ H264Result_t H264Depacketizer_GetFrame( H264DepacketizerContext_t * pCtx,
         }
     }
 
-    pFrame->frameDataLength = currentFrameDataIndex;
+    if( pFrame != NULL )
```

Comment:
```suggestion
    if( result == H264_RESULT_OK )
```
In user side, this frame data length is only used when it returns OK. I'd suggest to simply check the result becuase it's checked at BAD_PARAMS.